src/__swaps__/screens/Swap/components/CoinRow.tsx

```diff
@@ -111,7 +111,12 @@ export const CoinRow = ({
               <CoinRowButton icon="􀅳" outline size="icon 14px" />
               <CoinRowButton
                 color={isFavorite(address) ? '#FFCB0F' : undefined}
-                onPress={() => toggleFavorite(address)}
+                onPress={e => {
```

Comment:
Does this also work on Android? Mostly curious because I think I remember this not working for something I was working on forever ago.

Reply:
hmm rings a bell, i'll check it out